selfdrive/car/toyota/carcontroller.py

```diff
@@ -95,10 +96,13 @@ def update(self, enabled, active, CS, frame, actuators, pcm_cancel_cmd, hud_aler
       if pcm_cancel_cmd and CS.CP.carFingerprint in [CAR.LEXUS_IS, CAR.LEXUS_RC]:
```

Comment:
If `create_acc_cancel_command_2` works on all cars (the Lexi), we can replace `create_acc_cancel_command` and just do it here